### PR TITLE
Add about nested attributes

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,39 @@ $ tailf /var/log/td-agent/td-agent.log
...
+<filter kubernetes.**>
+  @type record_transformer
+  enable_ruby
+  <record>
+    kubernetes_namespace ${record["kubernetes"]["namespace"]}
+  </record>
+</filter>
+<match kubernetes.**>
+  @type rewrite_tag_filter
+  rewriterule1 namespace (.+) $1.${tag}
```
Comment (y-ken) [inline README.md]: do you mean to this?
s/namespace /kubernetes_namespace/

File: README.md [hunk 2]
```diff
@@ -116,6 +116,39 @@ $ tailf /var/log/td-agent/td-agent.log
...
+</match>
+```
+
+```
+{
+  "kubernetes": {
+    "namespace_name": "default"
```
Comment (y-ken) [inline README.md]: do you mean to this?
s/namespace_name /namespace/

**Comment by y-ken** [comment]
Thank you!

**Comment by y-ken** [comment]
It seems OK! Thank you!

**Comment by okkez** [comment]
Thank you for merging.

**Pull request merged**